src/gui/DatabaseWidget.cpp

```diff
@@ -497,7 +497,16 @@ void DatabaseWidget::openUrlForEntry(Entry* entry)
 
     if (urlString.startsWith("cmd://")) {
         if (urlString.length() > 6) {
-            QProcess::startDetached(urlString.mid(6));
+            QMessageBox::StandardButton result;
+            result = MessageBox::question(
+                this, tr("Execute command?"),
+                tr("Do you really want to execute the following command?<br><br>%1")
+                .arg(urlString),
```

Comment:
You should probably also truncate it if it is too long. Otherwise the message box will be gigantic for very long commands.